README.md

```diff
@@ -23,38 +23,110 @@ Ansible Role for installing the Docker runtime environment on your (Debian) host
 
 ## Requirements
 
-*nothing special*
+This is intended for Debian GNU/Linux based systems.
+Plain old Debian works, Ubuntu should work, others might work.
```

Comment:
"The role has been tested on Debian and is expected to work on Ubuntu. Other distribution have not been tested."